### ISSUE TITLE
Migration to 10.6 fails on Jellyfin.Server.Migrations.Routines.RemoveDuplicateExtras caused by permission issue

### ISSUE THREAD

**Issue by Spomyx** [jellyfin/jellyfin#3743]
**Describe the bug**
Upgrading jellyfin from 10.5.5 to 10.6.X on Docker fails with error 

[2020-07-29 16:11:56.676 +00:00] [INF] [1] Jellyfin.Server.Migrations.MigrationRunner: Applying migration '"RemoveDuplicateExtras"'
[2020-07-29 16:18:15.993 +00:00] [ERR] [1] Jellyfin.Server.Migrations.Routines.RemoveDuplicateExtras: Cannot make a backup of "library.db" at path "/config/data/library.db.bak1"
System.UnauthorizedAccessException: Access to the path is denied.

tried with 777 permission just to be sure but doesn't work either:
-rwxrwxrwx 1 root root  70M Jul 29 16:11 library.db.bak1

maybe the permission issue is on destination folder of the backup but i don't know what it is and i expect it to be handled by the provided docker image.

**System (please complete the following information):**
 - OS: Ubuntu arm
 - Virtualization: Docker
 - Clients: N/A
 - Browser: N/A
 - Jellyfin Version: 10.5.5 to 10.6.X
 - Playback: N/A
 - Installed Plugins: N/A
 - Reverse Proxy: N/A
 - Base URL: N/A]
 - Networking: N/A
 - Storage: SMB share

**To Reproduce**
1. Stop container with version 10.5.5
2. Change docker file to use 10.6.X
3. Start the container

**Expected behavior**
database backup is successful and migration goes on

**Logs**
```
[2020-07-29 16:11:35.297 +00:00] [INF] [1] Main: Jellyfin version: "10.6.1"
[2020-07-29 16:11:35.366 +00:00] [INF] [1] Main: Environment Variables: ["[JELLYFIN_SERVICE_PORT, 8096]", "[JELLYFIN_WIP_SERVICE_HOST, 10.107.211.85]", "[JELLYFIN_DATA_DIR, /config]", "[JELLYFIN_PORT_8096_TCP_ADDR, 10.101.215.140]", "[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_WIP_SERVICE_PORT, 8096]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_WIP_PORT_8096_TCP_PORT, 8096]", "[JELLYFIN_WIP_PORT, tcp://10.107.211.85:8096]", "[JELLYFIN_WIP_PORT_8096_TCP, tcp://10.107.211.85:8096]", "[JELLYFIN_WIP_SERVICE_PORT_WEB, 8096]", "[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_PORT_8096_TCP_PORT, 8096]", "[JELLYFIN_SERVICE_HOST, 10.101.215.140]", "[JELLYFIN_SERVICE_PORT_WEB, 8096]", "[JELLYFIN_WIP_PORT_8096_TCP_PROTO, tcp]", "[JELLYFIN_PORT_8096_TCP, tcp://10.101.215.140:8096]", "[JELLYFIN_WIP_PORT_8096_TCP_ADDR, 10.107.211.85]", "[JELLYFIN_MEDIA_DIR, /media]", "[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_PORT, tcp://10.101.215.140:8096]", "[JELLYFIN_PORT_8096_TCP_PROTO, tcp]", "[DOTNET_SYSTEM_GLOBALIZATION_INVARIANT, 1]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]"]
[2020-07-29 16:11:35.378 +00:00] [INF] [1] Main: Arguments: ["/jellyfin/jellyfin.dll"]
[2020-07-29 16:11:35.381 +00:00] [INF] [1] Main: Operating system: "Linux"
[2020-07-29 16:11:35.381 +00:00] [INF] [1] Main: Architecture: Arm64
[2020-07-29 16:11:35.384 +00:00] [INF] [1] Main: 64-Bit Process: True
[2020-07-29 16:11:35.384 +00:00] [INF] [1] Main: User Interactive: True
[2020-07-29 16:11:35.385 +00:00] [INF] [1] Main: Processor count: 4
[2020-07-29 16:11:35.385 +00:00] [INF] [1] Main: Program data path: "/config"
[2020-07-29 16:11:35.385 +00:00] [INF] [1] Main: Web resources path: "/jellyfin/jellyfin-web"
[2020-07-29 16:11:35.386 +00:00] [INF] [1] Main: Application directory: "/jellyfin/"
[2020-07-29 16:11:36.142 +00:00] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: "/cache"
[2020-07-29 16:11:36.179 +00:00] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[2020-07-29 16:11:38.930 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"AccessSchedule"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:38.930 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ActivityLog"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:38.931 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"ImageInfo"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:38.931 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"Permission"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:38.931 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"Preference"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:38.932 +00:00] [WRN] [1] Microsoft.EntityFrameworkCore.Model.Validation: The entity type '"User"' is configured to use schema '"jellyfin"'. SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[2020-07-29 16:11:56.676 +00:00] [INF] [1] Jellyfin.Server.Migrations.MigrationRunner: Applying migration '"RemoveDuplicateExtras"'
[2020-07-29 16:18:15.993 +00:00] [ERR] [1] Jellyfin.Server.Migrations.Routines.RemoveDuplicateExtras: Cannot make a backup of "library.db" at path "/config/data/library.db.bak1"
System.UnauthorizedAccessException: Access to the path is denied.
 ---> System.IO.IOException: Permission denied
   --- End of inner exception stack trace ---
   at System.IO.FileSystem.CopyFile(String sourceFullPath, String destFullPath, Boolean overwrite)
   at System.IO.File.Copy(String sourceFileName, String destFileName, Boolean overwrite)
   at Jellyfin.Server.Migrations.Routines.RemoveDuplicateExtras.Perform()
   at Jellyfin.Server.Migrations.MigrationRunner.Run(CoreAppHost host, ILoggerFactory loggerFactory)
```

**Comment by AlessandroSpa** [comment]
I have similar behaviour with update trough linuxserver.io docker image. Any ideas?

**Comment by Spomyx** [comment]
I gave up on that migration and created another docker from scratch on 10.6.1.
I tried to tweak my docker file in order to have the rffmpeg tool (https://github.com/joshuaboniface/rffmpeg) working in docker. Everything seemed ok on the network side of things but jellyfin couldn't read transcodes on NFS shares despite it being in 777 mode and running as root.
So i guess there's still some permissions issues with the docker image but it's beyond my knowledge.
